.circleci/config.yml

```diff
@@ -548,11 +573,24 @@ workflows:
               only: /^v(\d+\.)(\d+\.)(\d+)(.*)?$/
             branches:
               only: unsigned
+      - build-ios-iphonesimulator:
+          context: mattermost-mobile-unsigned
+          requires:
+            - test
+          filters:
+            tags:
+              only: /^v(\d+\.)(\d+\.)(\d+)(.*)?$/
+            branches:  
+              only:
+                - unsigned
```

Comment:
The unsigned branch is only created once final is released, so I don't think this works here

Reply:
Okay, I changed this to 
```yml
- /^build-\d+$/
- /^build-ios-\d+$/
- /^build-ios-beta-\d+$/
```